### PR TITLE
implement setup cross platform

### PR DESCRIPTION
**Pull request by congsonzero**

### CODE REVIEW

**Comment by congsonzero** [comment]
@ducquang2  help me

**Review by ducquang2 — COMMENTED** [review]
ok.

File: setup/setup.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+import subprocess,sys
```
Comment (ducquang2) [inline setup/setup.py:2]: pls drop a new line after line 2

**Review by ducquang2 — COMMENTED** [review]
review

**Pull request merged**